apps/README.md

```diff
@@ -0,0 +1,5 @@
+# apps
+
+The main source code for the PrairieLearn project can be found in the [`prairielearn`](prairielearn/) directory.
+
+The `grader-host` and `workspace-host` applications are used to run the grading and workspace containers.
```

Comment:
Might be worth commenting that the grader host includes the production version of the grading process, with the dev version running directly inside prairielearn by default; while the workspace host runs both in production and dev.

Reply:
How does 901eaeb look?